src/styles/App.css

```diff
@@ -0,0 +1,77 @@
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  border-bottom: 1px solid #e8e8e8;
+  padding: 20px 100px;
+  margin-bottom: 20px;
+}
+
+.header-leftside {
+  display: flex;
+  align-items: center;
+}
+
+.header-leftside > * {
+  margin-right: 60px;
+}
+
+.header-leftside span,
+.header-leftside select {
+  font-size: 15px;
+  border: none;
+  background-color: transparent;
+}
+
+header img {
+  width: 40px;
+  height: 40px;
+}
+
+.BooksList {
+  padding-bottom: 25px;
+  border-bottom: 2px solid #e8e8e8;
+}
+
+.form-container {
+  padding: 40px 100px;
+}
+
+.form-container .title {
+  color: #888;
+  font-size: 22px;
+  margin-bottom: 20px;
+}
+
+.BooksForm {
+  display: flex;
+  align-items: center;
```

Comment:
- [ ] You have these flex and center rules repeated 4 times, please create a CSS class just with them and apply them in the necessary elements in your HTML to avoid CSS bad practices.